sample_files/pre-commit-13.0/.pre-commit-config.yaml

```diff
@@ -0,0 +1,62 @@
+exclude: "^setup/"
+default_language_version:
+  python: python3
+repos:
+- repo: https://github.com/ambv/black
+  rev: 19.3b0
+  hooks:
+  - id: black
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.3.0
+  hooks:
+  - id: trailing-whitespace
+    # exclude autogenerated files
+    exclude: /README\.rst$|\.pot?$
+  - id: end-of-file-fixer
+    # exclude autogenerated files
+    exclude: /README\.rst$|\.pot?$
+  - id: debug-statements
+  - id: flake8
+    name: flake8 except __init__.py
+    exclude: /__init__\.py$
+    additional_dependencies: ["flake8-bugbear==19.8.0"]
+  - id: flake8
+    name: flake8 only __init__.py
+    args: ["--extend-ignore=F401"]  # ignore unused imports in __init__.py
+    files: /__init__\.py$
+    additional_dependencies: ["flake8-bugbear==19.8.0"]
+- repo: https://github.com/pre-commit/mirrors-pylint
+  rev: v2.3.1
+  hooks:
+  - id: pylint
+    name: pylint with optional checks
+    args: ["--rcfile=.pylintrc", "--exit-zero"]
+    verbose: true
+    additional_dependencies: ["pylint-odoo==3.0.3"]
+- repo: https://github.com/pre-commit/mirrors-pylint
```

Comment:
Why is this repo duplicated? It might be OK or not, I'm just curious...

Reply:
The first one is used for optional checks, the second one for mandatory checks

Reply:
Oh, but you mean, we could add the 2 hooks in the same `repo`. Probably?

Reply:
It works: https://github.com/acsone/maintainer-quality-tools/pull/2